src/melee/lb/lbcollision.h

```diff
@@ -1,24 +1,39 @@
 #ifndef MELEE_LB_LBCOLLISION_H
 #define MELEE_LB_LBCOLLISION_H
 
+#include <melee/ft/forward.h>
+#include <melee/lb/forward.h>
 #include <sysdolphin/baselib/jobj.h>
 
-/// @todo Does not belong here.
-typedef struct _Hitbox {
-    s32 x0;
+struct Hitbox {
+    Tangibility tangiblity;
```

Comment:
This is probably wrong, or at least the elements of `Tangibility` are named wrong, but having it be any enum is better than the alternative. It may or may not share an enum with `Hurtbox::tangibility`.